src/frontend/src/components/Posts/LoadAutoScroll.jsx

```diff
@@ -0,0 +1,58 @@
+import React, { useEffect } from 'react';
+import PropTypes from 'prop-types';
+
+import { Container, Button, Grid } from '@material-ui/core';
+import { makeStyles } from '@material-ui/core/styles';
+
+const useStyles = makeStyles((theme) => ({
+  content: {
+    '& > *': {
+      padding: theme.spacing(2),
+      bottom: theme.spacing(4),
+    },
+  },
+}));
+
+function LoadMoreButton({ onScroll }) {
+  const classes = useStyles();
+  const $buttonRef = React.useRef(null);
```

Comment:
Import `useRef` on line 1 too, please.

Reply:
Yep! I thought import the `useRef` would be nicer! I will do that!